### PR TITLE
fix(docker): Fixed entrypoint environment log-level and service-id cover by default value

### PR DESCRIPTION
**Pull request by Luckyboys**

### CODE REVIEW

File: run/docker/entrypoint.sh [hunk 1]
```diff
@@ -14,12 +14,12 @@
 ## limitations under the License.
 
 logLevel="${LOG_LEVEL}"
-if [ -n "${logLevel}" ]; then
+if [ -z "${logLevel}" ]; then
   logLevel="info"
 fi
```
Comment (ponypaver) [inline run/docker/entrypoint.sh]: logLevel=${LOG_LEVEL:-info}

File: run/docker/entrypoint.sh [hunk 2]
```diff
@@ -14,12 +14,12 @@
...
 
 serviceId="${SERVICE_ID}"
-if [ -n "${serviceId}" ]; then
+if [ -z "${serviceId}" ]; then
   serviceId="0"
 fi
```
Comment (ponypaver) [inline run/docker/entrypoint.sh]: serviceId=${SERVICE_ID:-0}

File: run/docker/entrypoint.sh (hunk 1 above)
Comment (Luckyboys) [reply run/docker/entrypoint.sh]: Fixed

File: run/docker/entrypoint.sh (hunk 2 above)
Comment (Luckyboys) [reply run/docker/entrypoint.sh]: Fixed

**Pull request merged**